source/Octopus.Tentacle.Tests.Integration/ClientScriptExecutionAdditionalScripts.cs

```diff
@@ -28,7 +28,7 @@ public async Task AdditionalScriptsWork(TentacleConfigurationTestCase tentacleCo
                 .CreateFile(path) // How files are made are different in bash and powershell, doing this ensures the client and tentacle really are using the correct script.
                 .Print("Hello");
 
-            var startScriptCommand = new StartScriptCommandV2Builder()
+            var startScriptCommand = new StartScriptCommandV3AlphaBuilder()
```

Comment:
💭 Just an idea, but since we've had to replace 52 instances of `StartScriptCommandV2Builder` with `StartScriptCommandV3AlphaBuilder`, did we want to instead make a new builder called something like `StartScriptCommandLatestBuilder` or `LatestStartScriptCommandBuilder`, and use that instead everywhere?

That way, when we do V4, it won't be as invasive.

`LatestStartScriptCommandBuilder` could extend `StartScriptCommandV3AlphaBuilder`, so we keep `StartScriptCommandV3AlphaBuilder` for when it is needed.

Anyway, just a thought 🙂

Reply:
Yeah, that's not a bad idea. I'll make that change 👍

Reply:
Addressed in [cf37208](https://github.com/OctopusDeploy/OctopusTentacle/pull/669/commits/cf3720897e4b34f2f1399e93609abad6fff26098)